Server/controllers/inviteController.js

```
@@ -1,101 +1,111 @@
-      .json({ success: true, msg: "Invite sent", data: inviteToken });
-  } catch (error) {
-    error.service === undefined ? (error.service = SERVICE_NAME) : null;
-    error.method === undefined ? (error.method = "inviteController") : null;
-    next(error);
-    return;
-  }
+        return res
+            .status(200)
+            .json({success: true, msg: "Invite sent", data: inviteToken});
+    } catch (error) {
+        error.service === undefined ? (error.service = SERVICE_NAME) : null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)